lib/build_targets/plugins.dart

```diff
@@ -133,17 +110,14 @@ class NativePlugins extends Target {
     final List<String> pluginClasses = <String>[];
 
     for (final TizenPlugin plugin in nativePlugins) {
-      // Create a copy of the plugin to allow editing its projectFile.
-      final TizenPlugin pluginCopy = plugin.copyWith(
-        directory: environment.fileSystem.systemTempDirectory.createTempSync(),
-      );
-      copyDirectory(plugin.directory, pluginCopy.directory);
-
-      _ensureStaticLibType(pluginCopy, environment.logger);
```

Comment:
Why don't you keep this check? If the plugin is a shared lib, it would be okay to cause an error.

Reply:
Okay I'll fix.

Reply:
https://github.com/flutter-tizen/flutter-tizen/pull/303/commits/d2a599bae55e2f86c813d48e0b3b5a2f3a4e2e01

Reply:
👍 thank you.